nilearn/_utils/fixes/__init__.py

```diff
@@ -10,11 +10,8 @@
     from sklearn.feature_selection import f_regression
 
 try:
-    if LooseVersion(sklearn.__version__) >= LooseVersion('0.18'):
-        from sklearn.model_selection import check_cv
     # 0.18 > scikit-learn >= 0.16
-    else:
-        from sklearn.cross_validation import check_cv
+    from sklearn.cross_validation import check_cv
```

Comment:
Ok, my bad (again).
The cross_validation module is going to be deprecated in 0.20 not 0.18.
The check_cv in model_selection does not receive X as param, as well as the CV API. Then, we have to think about this deprecation in near future --e.g. using cross validation in Nilearn examples.
For the moment we can still use the check_cv in cross_validation.